test/distributed/org/apache/cassandra/distributed/test/sai/SAIUtil.java

```diff
@@ -115,7 +115,7 @@ private static InetAddressAndPort nodeAddress(InetSocketAddress address)
     public static List<String> getIndexes(Cluster cluster, String keyspace)
     {
         waitForSchemaAgreement(cluster);
-        String query = String.format("SELECT index_name FROM system_views.sai_indexes WHERE keyspace_name = '%s' ALLOW FILTERING", keyspace);
+        String query = String.format("SELECT index_name FROM system_views.sai_column_indexes WHERE keyspace_name = '%s' ALLOW FILTERING", keyspace);
```

Comment:
nit: Might as well use `ColumnIndexesSystemView.NAME`?

Reply:
That would make sense. Dangers of changing things in a hurry :)